docs/Template.md

```diff
@@ -23,18 +23,21 @@ A template from a Cluster generator:
 ```
 
 The template subfields correspond directly to [the spec of an Argo CD `Application` resource](https://argoproj.github.io/argo-cd/operator-manual/declarative-setup/#applications):
+
 - `project` refers to the [Argo CD Project](https://argoproj.github.io/argo-cd/user-guide/projects/) in use (`default` may be used here to utilize the default Argo CD Project)
 - `source` defines from which Git repostory to extract the desired Application manifests
-  - **repoURL**: URL of the repository (eg `https://github.com/argoproj/argocd-example-apps.git`)
-  - **targetRevision**: Revision (tag/branch/commit) of the repository (eg `HEAD`)
-  - **path**: Path within the repository where Kubernetes manifests (and/or Helm, Kustomize, Jsonnet resources) are located 
+    - **repoURL**: URL of the repository (eg `https://github.com/argoproj/argocd-example-apps.git`)
+    - **targetRevision**: Revision (tag/branch/commit) of the repository (eg `HEAD`)
+    - **path**: Path within the repository where Kubernetes manifests (and/or Helm, Kustomize, Jsonnet resources) are located 
 - `destination`: Defines which Kubernetes cluster/namespace to deploy to
-  - **name**: Name of the cluster (within Argo CD) to deploy to
-  - **server**: API Server URL for the cluster (Example: `https://kubernetes.default.svc`)
-  - **namespace**: Target namespace in which to deploy the manifests from `source` (Example: `my-app-namespace`)
-  - Note:
-    - Referenced clusters must already be defined in Argo CD, for ApplicationSet controller to use them
-    - Only **one** of `name` or `server` may be specified: if both are specified, an error is returned.
+    - **name**: Name of the cluster (within Argo CD) to deploy to
+    - **server**: API Server URL for the cluster (Example: `https://kubernetes.default.svc`)
+    - **namespace**: Target namespace in which to deploy the manifests from `source` (Example: `my-app-namespace`)
+
+But note:
```

Comment:
"But notes" sounds a bit strange to me.   Drop the "But"?

Reply:
Done.